docs/buildkitd.toml.md

```diff
@@ -35,6 +35,9 @@ insecure-entitlements = [ "network.host", "security.insecure" ]
   debugAddress = "0.0.0.0:6060"
   uid = 0
   gid = 0
+  # this is for Windows scenarios, to give RW access to the named pipe to specific users
```

Comment:
Maybe lets skip this in here as for most people it would be confusing and Windows users should almost always use `--group`. We can mention this in `windows.md` if you like.